### ISSUE TITLE
array_key_exists() expects parameter 2 to be array

### ISSUE THREAD

**Issue by RavishaHesh** [pusher/push-notifications-php#14]
I get following error when sending a notification
```
ErrorException: array_key_exists() expects parameter 2 to be array, null given in /var/www/EtenNewWeb/vendor/pusher/pusher-push-notifications/src/PushNotifications.php:59
```

my code:
```php
public static function sendNotification($interest, $title, $body)
{
	try {
		$pushNotifications = new PushNotifications([
  			'instanceId' => env('PUSHER_PUSH_INSTANCE', null),
  			'secretKey' => env('PUSHER_PUSH_SECRET_KEY', null),
		]);

		$publishResponse = $pushNotifications->publish([$interest], [
			'apns' => [
				'aps' => [ 'alert' => $body ]
			],
			'fcm' => [
				'notification' => [
					'title' => $title,
					'body' => $body
				]
			]
		]);
	} catch (\Exception $e) {
		Log::error($e);
	}
}
```

**Update:**

Error was caused by an invalid `instance-id`. When I updated my code with correct `instance-id` it went away. But when `instance-id` is invalid it's easy if you can add a proper error message so it's easy to identify the error.
Thank You

**Comment by turingincomplete** [comment]
Hi! Thanks for the issue & PR!
I hope to have a fix for this released today. I'll update you when it's out.

**Comment by turingincomplete** [comment]
This should be fixed in v0.10.5

Thanks for the report! :+1:
